CHANGES.md

```diff
@@ -143,7 +143,8 @@
 
 ## Known Issues
 
-* Long-running Python pipelines might suffer from a memory leak: [#28246](https://github.com/apache/beam/issues/28246).
+* Long-running Python pipelines might experience a memory leak: [#28246](https://github.com/apache/beam/issues/28246).
```

Comment:
do you plan to update the blogs like website/www/site/content/en/blog/beam-2.49.0.md?

Reply:
good question, this escaped my attention.
I wonder if we should redirect known issues to CHANGES.md to avoid duplicating the content.
I'll update for now.